issue#94 : casting Windowspath to string. example/tutorials/convert_excel_to_mtg.ipynb

simonakasimon opened the pull request.

simonakasimon committed:
casting Windowspath to string

pradal commented:
I have developped a fix and merge it in the master/main.
Can you test it on Windows?

simonakasimon commented:
hello,

it is not working because of "AttributeError: 'WindowsPath' object has no attribute split"
maybe it could work using the basename function like :

```
names=[os.path.basename(filename) for filename in path]
```
instead of 

``` 
names=[filename.split(os.sep)[-1].split(".")[0] for filename in path]
``` 

but you will need to modify the naming of the files in the path_data function because the basename function keeps the extension in the file name. 

you can find out more here : https://stackoverflow.com/questions/8384737/extract-file-name-from-path-no-matter-what-the-os-path-format
